http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java

```diff
@@ -182,15 +183,19 @@ public void testExplicitPropertyMappings()
     }
 
     @Test
-    public void testInvalidHttpsConfiguration()
+    public void testHttpsConfigurationValid()
```

Comment:
```suggestion
    public void testHttpsConfigurationValidation()
```

(you're testing both valid and invalid cases)